src/handleCommand.ts

```diff
@@ -18,43 +18,59 @@
 
 import { spawn } from "child_process";
 import { readdirSync } from "fs";
+import path from "path";
 import { Handler, InbuiltCommand } from "./classes";
-import { ExitCodes } from "./constants";
-import { CommandResult } from "./types";
-import { findInPath } from "./util";
+import { ExitCodes } from "./util/constants";
+import { exists, findInPath, isExecutable } from "./util/fs";
+import { CommandResult } from "./util/types";
 
 export class CommandHandler implements Handler {
-  public static inbuiltCommands: Record<string, new () => InbuiltCommand> = Object.create(null);
+  public static inbuiltCommands: Record<string, new () => InbuiltCommand> = {};
```

Comment:
`Object.create(null)` actually removes all prototypial inheritance and makes it an absolutely clean object, but OK, I guess

Reply:
i changed the check from `in` to hasOwnProperty which ignores object inbuilts, so this can be reverted